src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -1151,6 +1191,29 @@ public Task DocumentResetAsync(Document document, CancellationToken cancellation
             #endregion
         }
 
+        private class Analyzer2 : IIncrementalAnalyzer
+        {
+            public readonly List<DocumentId> DocumentIds = new List<DocumentId>();
+
+            public Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, InvocationReasons reasons, CancellationToken cancellationToken)
+            {
+                this.DocumentIds.Add(document.Id);
+                return SpecializedTasks.EmptyTask;
+            }
+
+            #region unused 
+            public bool NeedsReanalysisOnOptionChanged(object sender, OptionChangedEventArgs e) => false;
```

Comment:
:memo: All of these should throw `ExceptionUtilities.Unreachable`

Reply:
it is reachable. will be called.